frontend/src/components/sign/components/sign-in-form/sign-in-form.tsx

```diff
@@ -80,6 +80,13 @@ export const SignInForm = (): React.ReactElement => {
             control={control}
             inputLabel="Password"
           />
+
+          <p className={styles.forgotPassword}>
+            <Link to={AppRoute.FORGOT_PASSWORD} className={styles.link}>
+              Forgot password?
+            </Link>
+          </p>
```

Comment:
Why did you create an additional `p` tag? Couldn't the `Link` be appropriately styled?

Reply:
just to reuse styles.link style

Reply:
We can combine `className`s with the `clsx` utility :slightly_smiling_face: 
What was the issue with the reusage?